code/modules/mob/living/carbon/human/species_types/apid.dm

```diff
@@ -17,11 +17,13 @@
 	mutanteyes = /obj/item/organ/eyes/apid
 	mutantlungs = /obj/item/organ/lungs/apid
 	mutantwings = /obj/item/organ/wings/bee
-	heatmod = 1.5
-	coldmod = 1.5
-	burnmod = 1.5
+	heatmod = 2
+	coldmod = 2
+	burnmod = 2
+	staminamod = 2
```

Comment:
Wew this one is bad, considering stamina is the primary combat mechanic. For example this makes stun batons a 1 shot kill

Reply:
changed